projects/admin/src/app/record/detail-view/circ-policy-detail-view/circ-policy-detail-view.component.html

```diff
@@ -22,148 +22,132 @@ <h1 class="mb-3">{{ data.name | translate }}</h1>
-          {{ 'Renewal duration' | translate }}:
-        </dt>
-        <dd class="col-sm-7 col-md-8 mb-0">
-          {{ data.renewal_duration }}
-        </dd>
-      </dl>
+    <section class="card mt-4">
+      <div class="card-header"x>
+        <b translate>Circulation settings</b>
+      </div>
+      <div class="card-body">
+        <!-- ALLOW CHECKOUT -->
+        <dl class="row mb-0">
+          <dt class="col-sm-3 offset-sm-2 offset-md-0 label-title" translate>Allow checkout</dt>
+          <dd class="col-sm-7 col-md-8 mb-0">
+            <ng-container *ngIf="data.allow_checkout; then thenCheckoutBlock else elseCheckoutBlock">
+            </ng-container>
+            <ng-template #thenCheckoutBlock>
+              <i id="cipo-allow-checkout" class="fa fa-check text-success" aria-hidden="true"></i>
+            </ng-template>
+            <ng-template #elseCheckoutBlock>
+              <i id="cipo-allow-checkout" class="fa fa-times text-danger" aria-hidden="true"></i>
+            </ng-template>
+          </dd>
+        </dl>
+        <!-- CHECKOUT DURATION -->
+        <dl class="row mb-0" *ngIf="data.checkout_duration || data.checkout_duration >= 0">
+          <dt class="col-sm-3 offset-sm-2 offset-md-0 label-title" translate>Checkout duration</dt>
+          <dd id="cipo-checkout-duration" class="col-sm-7 col-md-8 mb-0">
+            {{ data.checkout_duration }}
+          </dd>
+        </dl>
+        <!-- NUMBER OF RENEWALS -->
+        <dl class="row mb-0" *ngIf="data.number_renewals">
+          <dt class="col-sm-3 offset-sm-2 offset-md-0 label-title" translate>Number of renewals</dt>
+          <dd id="cipo-cnumber-renewals" class="col-sm-7 col-md-8 mb-0">
+            {{ data.number_renewals }}
+          </dd>
+        </dl>
+        <!-- RENEWAL DURATION -->
+        <dl class="row mb-0" *ngIf="data.renewal_duration">
+          <dt class="col-sm-3 offset-sm-2 offset-md-0 label-title" translate>Renewal duration</dt>
+          <dd id="cipo-renewal-duration" class="col-sm-7 col-md-8 mb-0">
+            {{ data.renewal_duration }}
+          </dd>
+        </dl>
+        <!-- ALLOW REQUESTS -->
+        <dl class="row mb-0">
+          <dt class="col-sm-3 offset-sm-2 offset-md-0 label-title" translate>Allow requests</dt>
+          <dd class="col-sm-7 col-md-8 mb-0">
+            <ng-container *ngIf="data.allow_requests; then thenRequestBlock else elseRequestBlock"></ng-container>
+            <ng-template #thenRequestBlock>
+              <i id="cipo-allow-request" class="fa fa-check  text-success" aria-hidden="true"></i>
+            </ng-template>
+            <ng-template #elseRequestBlock>
+              <i id="cipo-allow-request" class="fa fa-times text-danger" aria-hidden="true"></i>
+            </ng-template>
+          </dd>
+        </dl>
+        <!-- IS DEFAULT -->
+        <dl class="row mb-0">
+          <dt class="col-sm-3 offset-sm-2 offset-md-0 label-title" translate>Is default</dt>
```

Comment:
```suggestion
          <dt class="col-sm-3 offset-sm-2 offset-md-0 label-title" translate>Is default</dt>
```
Is there a possibility to display in a tooltip or something the following help message (description): *This circulation policy is applied by default, if no other circulation policy is defined* ?

Reply:
Yes of course